src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -4,18 +4,19 @@
 using AspNet.Security.OpenIdConnect.Server;
 using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http.Authentication;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using OpenIddict;
+using OpenIddict.Mvc;
+using Orchard.Mvc;
 using Orchard.OpenId.Services;
 using Orchard.OpenId.ViewModels;
 using Orchard.Users.Models;
 
 namespace Orchard.OpenId.Controllers
 {
-    [Authorize, ValidateAntiForgeryToken]
+    [Authorize, AutoValidateAntiforgeryToken]
```

Comment:
Isn't it supposed to be applied at global level only, which is already done? Or is that an extraneous safety level to ensure it's really there. But then the previous one would be fine too.

https://github.com/aspnet/Announcements/issues/156